src/app/components/popover/index.html

```diff
@@ -1,13 +1,161 @@
-<sky-demo-page
+<sky-tabbed-demo-page
   pageTitle="Popover"
-  skyuxModule="SkyPopoverModule"
-  packageName="@skyux/popovers"
 >
 
   <sky-demo-page-summary>
     The popover module allows you to display contextual content when users interact with an element on a page. The content can be plain text, HTML, or an Angular component.
   </sky-demo-page-summary>
+	
+	<sky-demo-page-example>
+    <sky-popover-demo>
+    </sky-popover-demo>
+  </sky-demo-page-example>
 
+<sky-tabset id="content-tabs">
+	
+	<sky-tab
+		tabHeading="Design">
+	
+		<sky-demo-page-content sectionHeading="Usage">
+			<h3>Use when</h3>
+				<p>You need to give users access to small pieces of contextual content that are not always valuable to see.</p>
+				<p>You need to guide users toward subtle-but-important elements when tasks include complex or unfamiliar capabilities.</p>
+			<h3>Don't use when</h3>
+				<p>The content is valuable to see most of the time. Use <a routerLink="/design/guidelines/user-assistance" fragment="persistent-inline-help">presistent inline help</a> instead.</p>
+				<p>You need to show a large amount of content. Use a <a routerLink="/components/flyout">flyout</a> instead.</p>
+		</sky-demo-page-content>
+		
+		<sky-demo-page-content sectionHeading="Anatomy">
+			<div class="two-col-right-image sky-margin-stacked-separate">
+			<div class="two-col-text">
+				<div class="anatomy-list">
+					<div class="anatomy-item anatomy-item-required"><div class="anatomy-marker">1</div><div class="anatomy-description">Content area</div></div>
+					<div class="anatomy-item anatomy-item-required"><div class="anatomy-marker">2</div><div class="anatomy-description">Highlight border</div></div>
+					<div class="anatomy-item anatomy-item-required"><div class="anatomy-marker">3</div><div class="anatomy-description">Element pointer</div></div>
+				</div>
+			</div>
+			<div class="two-col-image">
+				<div class="screenshot">
+					<img src="~/assets/img/guidelines/popover/anatomy.png" />
+				</div>
+			</div>
+		</div>
+		</sky-demo-page-content>
+		
+		<sky-demo-page-content sectionHeading="Options">
+			<h3>Positioning</h3>
+			<div class="two-col-right-image sky-margin-stacked-separate">
+				<div class="two-col-text">
+					<p>You can specify the position of the popover in relation to the element it is pointing to. The default position is above and centered on the element.</p>
+				</div>
+				<div class="two-col-image">
+					<div class="screenshot">
+						<img src="~/assets/img/guidelines/popover/option-position.png" />
```

Comment:
Should the image have a caption? I don't think so because it **seems** like the pattern we are following is to provide captions only when the images illustrate something to do or not to do. But I can't tell for certain. Is my assumption about the pattern correct?

Reply:
yeah, for the most part we'll only need captions for do's/don'ts

Reply:
Sounds good.